docs/tutorials/projects/helmet.md

```diff
@@ -0,0 +1,455 @@
+        <a href="https://app.viam.com/data/view?view=images">\
+        the DATA tab</a of the Viam app>.'
+    )
+
+    # Get a JSON-ready representation of the Mail object
+    mail_json = message.get()
+
+    # Send an HTTP POST request to /mail/send
+    response = sg.client.mail.send.post(request_body=mail_json)
+
+    return 'Notification sent.'
+```
+
+Click **DEPLOY**.
+
+For simplicity while developing and testing your code, you can [allow unauthenticated HTTP function invocation](https://cloud.google.com/functions/docs/securing/managing-access-iam#allowing_unauthenticated_http_function_invocation), though we recommend that you ultimately set up proper authentication for better security.
+
+#### Test the cloud function
+
+Now you can test the script:
+
+1. Navigate to the **TRIGGER** tab in the Google Cloud console.
+
+2. Click the copy button to copy the trigger URL.
+   Paste it into a new browser window and hit your **Enter** key.
+   The browser should display "Notification sent."
+   If it displays an error, double-check your code, and double-check your [access authorization](https://cloud.google.com/functions/docs/securing/managing-access-iam).
+
+3. Check your email inbox.
+
+   {{<imgproc src="/tutorials/helmet/email-received.png" resize="x600" declaredimensions=true alt="The email, opened in a web email client." style="max-width: 400px" >}}
+
+   If you don't see an email, check your spam folder.
+   If you still don't see an email, make sure your SendGrid account is fully set up (2FA enabled, email confirmed) and that your email API key is [correctly configured](#configure-email-credentials).
+
+### Configure a webhook on your machine
+
+Now it's time to configure a [webhook](/build/configure/#webhooks) on your machine to trigger the email cloud function when a person is not wearing a hard hat.
+Since you configured data to sync only when an image of a person without a hard hat is captured, configuring the webhook to trigger each time an image is synced to the cloud will produce the desired result.
+
+Configure a webhook as follows:
+
+1. Navigate to the **Config** tab of your machine.
+   Toggle the **Mode** to **Raw JSON**.
+
+2. Paste the following JSON template into your raw JSON config.
+   `"webhooks"` is a top-level section like `"components"`, `"services"`, or any of the other config sections.
+
+   ```json {class="line-numbers linkable-line-numbers"}
+     "webhooks": [
+       {
+         "url": "<Insert your own cloud function URL>",
+         "event": {
+           "attributes": {
+             "data_types": ["binary"]
+           },
+           "type": "part_data_ingested"
+         }
+       }
+     ]
+   ```
+
+3. Replace the `url` value with your cloud function URL.
+   You can get this URL by copying it from the **TRIGGER** tab in the cloud function console.
+   Once you've done this, the `url` line should resemble, for example, `"url": "https://us-east1-blah-blah-123456.cloudfunctions.net/hat-email"`.
```

Comment:
```suggestion
   Once you've done this, the `url` line should resemble `"url": "https://us-east1-abcd-efgh-123456.cloudfunctions.net/hat-email"`.
```

Reply:
I included "for example" because sometimes when we say something should resemble something, we really mean it should be an exact match. Fine to replace the blah blah part but that part is actually words, not random characters, so would this be okay?
```suggestion
   Once you've done this, the `url` line should resemble, for example, `"url": "https://us-east1-example-string-123456.cloudfunctions.net/hat-email"`.
```

Reply:
yep sounds good - just don't like the "blah" "blah". That's maybe a me-thing but it feels a bit unprofessional 🙈

Reply:
That's fair XD